New buffer size warning in UT assert

**Describe the bug**
After moving the UtAssert_DoTestSegmentReport() function, it has triggered a new warning when building with optimization enabled (BUILDTYPE=release).

```
/home/joe/code/cfecfs/github-cfs-bundle/osal/ut_assert/src/utassert.c:82:20: error: ‘%-20s’ directive output may be truncated writing between 20 and 127 bytes into a region of size between 117 and 125 [-Werror=format-truncation=]
   82 |              "%02u %-20s TOTAL::%-4u  PASS::%-4u  FAIL::%-4u   MIR::%-4u   TSF::%-4u   N/A::%-4u\n",
```

**To Reproduce**
Build with GCC9+ and `BUILDTYPE=release` flag.

**Expected behavior**
Should build cleanly.

**System observed on:**
Ubuntu 20.04

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/README.md
+++ b/README.md
@@ -155,7 +155,7 @@ UT Hook functions now have the capability to get argument values by name, which 
 
 - Minor updates (see <https://github.com/nasa/osal/pull/264>)
 
-### **_OFFICIAL RELEASE: 5.0.0_**
+### **_OFFICIAL RELEASE: 5.0.0 - Aquila_**
 
 - Changes are detailed in [cFS repo](https://github.com/nasa/cFS) release 6.7.0 documentation
 - Released under the Apache 2.0 license

--- a/ut_assert/src/utassert.c
+++ b/ut_assert/src/utassert.c
@@ -42,7 +42,7 @@
 UtAssert_CaseType_t DefaultContext = UTASSERT_CASETYPE_FAILURE;
 UtAssert_TestCounter_t UT_SegmentCounters = { 0 };
 UtAssert_TestCounter_t UT_TotalCounters = { 0 };
-static char CurrentSegment[128];
+static char CurrentSegment[64];
 
 /*
  * Function Definitions
@@ -76,7 +76,7 @@ void UtAssert_DoReport(const char *File, uint32 LineNum, uint32 SegmentNum, uint
 
 void UtAssert_DoTestSegmentReport(const char *SegmentName, const UtAssert_TestCounter_t *TestCounters)
 {
-    char ReportBuffer[128];
+    char ReportBuffer[144];
 
     snprintf(ReportBuffer, sizeof(ReportBuffer),
              "%02u %-20s TOTAL::%-4u  PASS::%-4u  FAIL::%-4u   MIR::%-4u   TSF::%-4u   N/A::%-4u\n",

